geoips/plugins/modules/readers/ssmi_binary.py

```diff
@@ -278,24 +282,24 @@ def fill_arrays():
     LORES = 64  # pixels per lo-res scan
     HIRES = 128  # pixels per gi-res scan
     MAXSCANS = 3000  # max lo-res scans per file
-    SCANTIME = 3.798  # approximate A-B scan interval
+    SCANTIME = 3.798  # approximate A-B scan interval  # NOQA
 
-    TRUE = 1
-    FALSE = 0
-    BUFSIZE = 4444
-    FRAMESIZE = 12798
-    FILLER = 0xA5
+    TRUE = 1  # NOQA
+    FALSE = 0  # NOQA
+    BUFSIZE = 4444  # NOQA
+    FRAMESIZE = 12798  # NOQA
+    FILLER = 0xA5  # NOQA
     EOF_LEN = 6
 
     # Return Codes
-    OK = 0
-    BAD_HDRS = 3
-    BAD_EOF = 4
+    OK = 0  # NOQA
+    BAD_HDRS = 3  # NOQA
+    BAD_EOF = 4  # NOQA
     BAD_LEN = 5
-    WRITE_ERR = 6
-    END_FILE = 7
-    FATAL_ERR = 8
-    CANT_OPEN = 9
+    WRITE_ERR = 6  # NOQA
+    END_FILE = 7  # NOQA
+    FATAL_ERR = 8  # NOQA
+    CANT_OPEN = 9  # NOQA
```

Comment:
Ie, these can most likely be  commented out (unless they are actually used in the read line)

Reply:
Just did that